docs/100_producten/100_borgstelling/borgstelling-uitbetalen.feature

```diff
@@ -55,7 +55,6 @@ Functionaliteit: Borgstelling uitbetalen
       | AFGEWEZEN             |
       | AANVRAAG_BEEINDIGD    |
       | BEEINDIGD             |
-      | UITBETAALD            |
```

Comment:
En wat gebeurt er nu wanneer je een uitbetaling verzoekt terwijl de borgstelling al is uitbetaald?

Reply:
Dan wordt er een bericht uitbetaald verstuurd met het originele document, zoals ook gedefinieerd in het process.

![Screenshot 2023-06-08 at 08 11 42](https://github.com/sbnnl/documentatie/assets/78080998/fbd3eea3-d668-421a-865d-9f001922c1b8)

Reply:
Top!